src/content/structured/components/tooltips/code.mdx

```diff
@@ -126,3 +142,52 @@ export const snippetsRight = [
     </IcButton>
   </IcTooltip>
 </ComponentPreview>
+
+### Chip
+
+export const snippetsDefault = [
+  {
+    language: "Web component",
+    snippet: `<ic-tooltip target="small-chip" label="This is a description"> 
+  <ic-chip id="small-chip" label="Small" size="small">
+    <svg slot="icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+      <path d="M10 0C4.48 0 0 4.48 0 10C0 15.52 4.48 20 10 20C15.52 20 20 15.52 20 10C20 4.48 15.52 0 10 0ZM10 3C11.66 3 13 4.34 13 6C13 7.66 11.66 9 10 9C8.34 9 7 7.66 7 6C7 4.34 8.34 3 10 3ZM10 17.2C7.5 17.2 5.29 15.92 4 13.98C4.03 11.99 8 10.9 10 10.9C11.99 10.9 15.97 11.99 16 13.98C14.71 15.92 12.5 17.2 10 17.2Z" />
+    </svg>
+  </ic-chip>
+</ic-tooltip>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcTooltip target="small-chip" label="This is a description">
+  <IcChip id="small-chip" label="Small" size="small">
+    <svg slot="icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Can this be updated to use `<SlottedSVG slot="icon" width="20" height="20" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg">`.
The reason for this is if a developer copies this snippet into a typescript project, the `slot` attribute on the SVG tag will display an error. Thanks